e2e/cypress/integration/notifications/desktop_notifications_spec.js

```diff
@@ -56,6 +56,37 @@ describe('Desktop notifications', () => {
         });
     });
 
+    it('MM-T482 Desktop Notifications - (at) here not rec\'d when logged off', () => {
+        cy.apiCreateUser({}).then(({user}) => {
+            cy.apiAddUserToTeam(testTeam.id, user.id);
+            cy.apiLogin(user);
```

Comment:
Please add visit to a channel after logged in so we can test this in isolation during maintenance by doing `it.only` (if needed).